hri_safety_sense/src/VscProcess.hpp

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
As we discussed, this should be moved into an include directory.  That way it can be composed.

Reply:
d580097